iset.mm

```diff
@@ -86457,6 +86630,46 @@ finite and infinite sets (and therefore if the set size is a nonnegative
     ( cxnn0 wcel cn0 wn cpnf wceq wo wi elxnn0 pm2.53 sylbi imp ) ABCZADCZEZAFG
     ZNOQHPQIAJOQKLM $.
 
+  ${
+    $d f i $.
+    $( The point at infinity in ` NN+oo ` .  (Contributed by Jim Kingdon,
```

Comment:
Maybe for clarity, and @avekens's remark shows it is necessary:
> The point at infinity in NN+oo is the constant sequence equal to 1o.

Reply:
Yeah, figuring out a way to mention "the constant sequence equal to 1o" does sound good to me. You have to be kind of deep into set.mm/iset.mm conventions to see `( A X. { B } )` and immediately think "constant function".

I went with

```
    $( The point at infinity in ` NN+oo ` (the constant sequence equal to
       ` 1o ` ).  (Contributed by Jim Kingdon, 14-Jul-2022.) $)
```